src/lib/routes/admin-api/environments-controller.ts

```diff
@@ -32,62 +31,42 @@ export class EnvironmentsController extends Controller {
     }
 
     async getAll(req: Request, res: Response): Promise<void> {
-        try {
-            const environments = await this.service.getAll();
-            res.status(200).json({ version: 1, environments });
-        } catch (e) {
-            handleErrors(res, this.logger, e);
-        }
+        const environments = await this.service.getAll();
+        res.status(200).json({ version: 1, environments });
```

Comment:
Love the new controller cleanup. So much easier to read and write :+1:

Reply:
jup, have been thinking abut this for a while, but was a bit tricky to get the typing's correct with express and typescript.